lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -380,8 +376,8 @@ defmodule RealtimeWeb.RealtimeChannel do
       {:error, :missing_claims} ->
         shutdown_response(socket, "Fields `role` and `exp` are required in JWT")
 
-      {:error, :expected_claims_map} ->
-        shutdown_response(socket, "Token claims must be a map")
+      {:error, :token_malformed} ->
+        shutdown_response(socket, "The token provided is not a valid JWT token")
```

Comment:
pet peeve "jason web token token"

```suggestion
        shutdown_response(socket, "The token provided is not a valid JWT")
```